pom.xml

```diff
@@ -291,7 +291,7 @@
         <configuration>
           <serverId>ossrh</serverId>
           <nexusUrl>https://oss.sonatype.org/</nexusUrl>
-          <autoReleaseAfterClose>false</autoReleaseAfterClose>
+          <autoReleaseAfterClose>true</autoReleaseAfterClose>
```

Comment:
With this change we should remove `mvn nexus-staging:release` from `release.sh`.